app/assets/stylesheets/components/_add-item.scss

```diff
@@ -68,3 +68,7 @@
     text-align: right;
   }
 }
+
+.remove_default_choose_file_button {
```

Comment:
I wonder if since this is just one declaration we could create a utility class instead? What do you think of something like...

```
.is-hidden {
  display: none;
}
```